rubocop.yml

```diff
@@ -1,3 +1,5 @@
+<% rubocop_version = Gem.loaded_specs.fetch("rubocop").version %>
```

Comment:
Is this the version on the bundle or in the system?

Reply:
It looks like the version of whatever the process is running, depending on if `bundle exec` (or equivalent binstub) is used or not.


#### Added logging and binstub
```console
% git diff -U1
diff --git a/rubocop.yml b/rubocop.yml
index aab03ce..99bae8c 100644
--- a/rubocop.yml
+++ b/rubocop.yml
@@ -1,2 +1,4 @@
 <% rubocop_version = Gem.loaded_specs.fetch("rubocop").version %>
+<% puts "RUBOCOP VERSION:          #{rubocop_version}" %>
+<% puts "RUBOCOP-MINITEST VERSION: #{Gem.loaded_specs["rubocop-minitest"]&.version}" %>
% bundle binstubs rubocop
```

#### `bundle info`
```console
% bundle info rubocop | grep '*'
  * rubocop (1.57.1)
% bundle info rubocop-minitest
Could not find gem 'rubocop-minitest'.
```

#### `gem info rubocop`
```console
% bundle exec gem info rubocop | grep '^rubocop'
rubocop (1.57.1)
% gem info rubocop | grep '^rubocop' 
rubocop (1.57.2, 1.57.1, 1.56.4, 1.56.3, 1.56.2, 1.56.1, 1.55.1, 1.54.2, 1.54.1, 1.52.1, 1.52.0, 1.50.2, 1.50.0, 1.49.0, 1.30.1)
```

#### `gem info rubocop-minitest`
```console
% bundle exec gem info rubocop-minitest | grep '^rubocop'
% gem info rubocop-minitest | grep '^rubocop' 
rubocop-minitest (0.32.2, 0.31.0)
```

#### Running `rubocop`
```console
% bundle exec rubocop | grep VERSION
RUBOCOP VERSION:          1.57.1
RUBOCOP-MINITEST VERSION: 
% bin/rubocop | grep VERSION
RUBOCOP VERSION:          1.57.1
RUBOCOP-MINITEST VERSION: 
% rubocop | grep VERSION
RUBOCOP VERSION:          1.57.2
RUBOCOP-MINITEST VERSION: 
```